src/Adapter/Module/AdminModuleDataProvider.php

```diff
@@ -219,6 +221,8 @@ public function setActionUrls(ModuleCollection $modules, ?string $specific_actio
             }
 
             // Let's filter the actions depending on conditions the module is in
+            // Hide upload action
+            unset($urls['upload']);
```

Comment:
Is it really needed to add the upload action in this service if in the end it is removed later? Maybe we don't need to update this service? 🤔

Reply:
Now you can remove this